2022_Recipe1

```diff
@@ -0,0 +1,37 @@
+File: Recipe 1 - Home Made Unique Crispy Chicken Recipe
+Type: Documentation
+Author: upxksha
+
+Follow the steps accordingly for the most satisfying results.
+
+Ingredients List 1: 
+Chicken 1kg
+Vinegar
+Onion
+Black Pepper
+Chilli Powder
+Chilli Flakes
+Tomato Sauce
+
+Ingredients List 2:
+Flour
+Corn Flour
+Black Pepper
+Onion
+Chilli Powder
+Chilli Flakes
+
+Step 1: 
+Mix the Ingredients List 1 well and let it marinate for about 1-2 hours.
```

Comment:
Is this marinating the chicken? Should the chicken be in large pieces, small pieces, bone-in, boneless. Recommendations on white or dark meat?